src/main/scala/mesosphere/marathon/api/v2/validation/AppValidation.scala

```diff
@@ -537,4 +551,7 @@ object AppValidationMessages {
 
   val DockerEngineLimitedToSingleContainerNetwork =
     "may only specify a single container network when using the Docker container engine"
+
+  val HealthCheckIpProtocolLimitation =
+    "Health check ipProtocol can only be used for docker container and mesos http/https/tcp healthcheck"
```

Comment:
I think this should now be "IPv6 can only be used for container type DOCKER and Mesos http/https/tcp health checks"